package.json

```diff
@@ -15,5 +15,8 @@
   "homepage": "https://github.com/bestander/html-minify-loader",
   "dependencies": {
     "minimize": "^1.8.1"
+  },
+  "devDependencies": {
```

Comment:
We did not test it with webpack 1. Since the loader only runs at compile time this seems like the best place for it for use with this loader.
If there are issues with it being in devDependencies it can be moved up.